plugins/Morpheus/templates/layout.twig

```diff
@@ -42,6 +42,8 @@
             {% endblock %}
         </div>
 
+        <div id="sizewarning" style="display: none;">This page can't be rendered on screens with width or height below 200px.</div>
```

Comment:
Can we move that to the `admin.twig`?

Placing it inside `block body` next to the `parent()` call should work. Or would that mean to place it in multiple templates because there are relevant instances of `#content.admin` not created by the admin template?

Reply:
Should the warning also be translatable?